source/_docs/guides/sendgrid.md

```diff
@@ -80,8 +80,8 @@ A stable release for Drupal 8 is not yet available for the [SMTP Authentication
 4. Use the following within SMTP Server Settings:
 
  **SMTP server**: smtp.sendgrid.net <br>
- **SMTP port**: 465<br>
- **Use encrypted protocol**: Use SSL<br>
+ **SMTP port**: 2525<br>
+ **Use encrypted protocol**: Either No Encryption or TLS<br>
```

Comment:
@garydylina my thought on it was to head off questions of "can I just leave it as no encryption?" But seeing as we didn't see that with it saying Use SSL before, Use TLS seems fine - and we can update it later if there are problems/questions coming in